src/vote.js

```diff
@@ -0,0 +1,28 @@
+let vote = class {
+  constructor() {}
+
+  async BlockCount(provider) {
+    console.log("Blockchain: " + (await provider.getBlockNumber()));
+  }
+
+  async PollCurrentStandings(pollSigner) {
+    let res = await pollSigner.currentStandings();
+    console.log("Current standings: " + res);
+  }
+
+  async PollVoteCount(pollSigner) {
+    let res = await pollSigner.voteCount();
+    console.log("Poll Vote Count: " + res);
+  }
+
+  async PollVote(pollSigner, vote) {
```

Comment:
When calling this functions, the parameter should prob be the public key of a wallet and than sign it inside the function.